rules/falco_rules.yaml

```diff
@@ -3056,6 +3056,24 @@
   priority: WARNING
   tags: [container, cis, mitre_lateral_movement]
 
+- macro: consider_userfaultfd_activities
+  condition: (always_true)
+
+- list: user_known_userfaultfd_processes
+  items: []
+
+- rule: Unprivileged Delegation of Page Faults Handling to a Userspace Process
+  desc: Detect a successful unprivileged userfaultfd syscall which might act as an attack primitive to exploit other bugs
+  condition: >
+    consider_userfaultfd_activities and
```

Comment:
Are we going to see some warning message thrown by Falco since the condition not started with `evt.type` filter.

Reply:
Hey @Kaizhe nope! :)

The Falco engine emits a warning IF an `evt.type` filter is missing at all, not if before it there's a macro. In fact, we have various rules using this pattern: a macro acting as a gate (value `never_true` or `always_true`) before the filter on the event type.

Reply:
Thanks!